HW_1/task_3.py

```diff
@@ -0,0 +1,31 @@
+"""
+3.Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости,
+в которой находится эта точка (или на какой оси она находится).
+
+Ввод: два значения типа <int>
+Вывод: значение типа <int> либо значение типа <str>
+
+Пример: 34/-30/4  2/4/1  -34/0/Точка на отрицательной части оси абсцисс
+"""
+
+x = int(input("Введите кoординату точки по оси X "))
+y = int(input("Введите кoординату точки по оси Y "))
+
+if (x > 0 and y > 0):
```

Comment:
Все условия, где есть `and` и какой то повторяющийся символ можно свернуть относительно этого символа, например.
```suggestion
if x > 0 < y:
```
И скобки в условиях не всегда нужны, это как раз тот самый случай.